lib/blockscore/response.rb

```diff
@@ -16,13 +16,13 @@ def handle_response(resource, response)
     def api_error(response)
       case response.code
       when 400
-        fail InvalidRequestError.new(response)
+        raise InvalidRequestError.new(response)
```

Comment:
Yeah we still do prefer `fail` I'm not sure why rubocop changed the default. @dkubb or @tjchambers do you see any value in only using `raise`?